examples/timer_subscribe.rs

```diff
@@ -3,11 +3,11 @@
 use core::fmt::Write;
 use futures::future;
 use libtock::console::Console;
-use libtock::result::TockResult;
 use libtock::timer;
 use libtock::timer::Duration;
 
-async fn main() -> TockResult<()> {
+#[libtock::main]
+async fn main() -> libtock::result::TockResult<()> {
```

Comment:
Could you restore the `use libtock::result::TockResult` part?

Reply:
Done.